YYCache/YYKVStorage.m

```diff
@@ -429,11 +442,13 @@ - (NSData *)_dbGetValueWithKey:(NSString *)key {
         const void *inline_data = sqlite3_column_blob(stmt, 0);
         int inline_data_bytes = sqlite3_column_bytes(stmt, 0);
         if (!inline_data || inline_data_bytes <= 0) return nil;
+        sqlite3_reset(stmt);
```

Comment:
sqlite3_reset(stmt); should add before line:444 if (!inline_data || inline_data_bytes <= 0) return nil;